config/build_ornl_cades.sh

```diff
@@ -37,47 +37,50 @@ CMAKE_FLAGS="-DCMAKE_C_COMPILER=mpicc \
              -DCMAKE_C_FLAGS=-xCOMMON-AVX512 \
              -DCMAKE_CXX_FLAGS=-xCOMMON-AVX512"
 
-# Configure and build cpu real AoS
+# Configure and build cpu real SoA
 echo ""
 echo ""
 echo "building QMCPACK for cpu AoS real for CADES SHPC Condo"
-mkdir -p build_cades_cpu_real
-cd build_cades_cpu_real
-cmake -DENABLE_SOA=0 $CMAKE_FLAGS ..
+mkdir -p build_cades_cpu_real_SoA
+cd build_cades_cpu_real_SoA
+cmake -DENABLE_SOA=1 $CMAKE_FLAGS ..
 make -j 16
 cd ..
-ln -sf ./build_cades_cpu_real/bin/qmcpack ./qmcpack_cades_cpu_real
+ln -sf ./build_cades_cpu_real_SoA/bin/qmcpack ./qmcpack_cades_cpu_real_SoA
 
-# Configure and build cpu complex AoS
+# Configure and build cpu complex SoA
 echo ""
 echo ""
 echo "building QMCPACK for cpu AoS complex for CADES SHPC Condo"
-mkdir -p build_cades_cpu_comp
-cd build_cades_cpu_comp
-cmake -DENABLE_SOA=0 -DQMC_COMPLEX=1 $CMAKE_FLAGS ..
+mkdir -p build_cades_cpu_comp_SoA
+cd build_cades_cpu_comp_SoA
+cmake -DENABLE_SOA=1 -DQMC_COMPLEX=1 $CMAKE_FLAGS ..
 make -j 16
 cd ..
-ln -sf ./build_cades_cpu_comp/bin/qmcpack ./qmcpack_cades_cpu_comp
+ln -sf ./build_cades_cpu_comp_SoA/bin/qmcpack ./qmcpack_cades_cpu_comp_SoA
 
-# Configure and build cpu real SoA
+CMAKE_FLAGS="-DCMAKE_C_COMPILER=mpicc \
+             -DCMAKE_CXX_COMPILER=mpicxx"
+
+# Configure and build cpu real SoA without AVX
 echo ""
 echo ""
 echo "building QMCPACK for cpu SoA real for CADES SHPC Condo"
-mkdir -p build_cades_cpu_real_SoA
-cd build_cades_cpu_real_SoA
+mkdir -p build_cades_cpu_real_SoA_noavx
```

Comment:
Better to name as "_batch" rather than "_SoA_noavx".

Reply:
Just to confirm, remove both "SoA" and "noavx" qualifiers and add "batch" ?

Reply:
In this case, you still get avx2. noavx512 is more accurate

Reply:
Remove all qualifiers here.